gloo/docs/installation/gateway/kubernetes/_index.md

```diff
@@ -1,193 +1,85 @@
 ---
 
-If this is your first time running Gloo, you’ll need to download the command-line interface (CLI) called `glooctl`
-on to your local machine. You’ll use this CLI to interact with Gloo, including installing it onto your Kubernetes cluster.
+## Install command line tool (CLI)
 
-### Options to Install Gloo
+The `glooctl` command line provides useful functions to install, configure, and debug Gloo, though it is not required to use Gloo.
 
-To install Gloo, you can use one of two options:
+To install `glooctl` using the [Homebrew](https://brew.sh) package manager, run the following.
 
-* [Install via the Command Line Interface (CLI) `glooctl` (recommended)](#install-gloo-via-command-line-interface-cli)
-* [Install via Helm](#install-gloo-with-helm)
-
-### Verify Installation
-
-* [Verify installation](#verify-your-installation)
-
-### Uninstall Gloo
-
-* [Uninstall Gloo](#uninstall)
-
-We highly recommend installing Gloo using the Gloo CLI as it simplifies a lot of the user experience of using Gloo.
-For power users, feel free to use the underlying `yaml` configuration files directly.
-
-These directions assume you've prepared your Kubernetes cluster appropriately. Full details on setting up your
-Kubernetes cluster [here](../setup_kubernetes).
-
-There are several options for deploying Gloo, depending on your use case and deployment platform.
-
-* **Gateway**: (**recommended**) Gloo's full feature set is available via its v1/Gateway API. The Gateway API
-is modeled on Envoy's API with the use of opinionated defaults to make complex configurations possible
-while maintaining simplicity when required.
-
-* **Ingress**: Gloo supports the Kubernetes Ingress resource, acting as a Kubernetes Ingress Controller.
-
-{{% notice note %}}
-ingress objects must have the annotation `"kubernetes.io/ingress.class": "gloo"` to be processed by the Gloo Ingress.
-{{% /notice %}}
-
-* **Knative**: Gloo integrates automatically with Knative as a cluster-level ingress for
-[*Knative-Serving*](https://github.com/knative/serving). Gloo can be used in this way as a lightweight replacement
-for Istio when using Knative-Serving.
-
-{{% notice info %}}
-If this process does not work, please [open an issue](https://github.com/solo-io/gloo/issues/new).
-We are happy to answer questions on our [diligently staffed Slack channel](https://slack.solo.io/) as well.
-{{% /notice %}}
-
----
-
-## Install Gloo via Command Line Interface (CLI)
-
-Choosing a deployment option for installing Gloo into your Kubernetes cluster:
+```shell
+brew install solo-io/tap/gloo
+```
 
-* [Gateway](#install-the-gloo-gateway-to-your-kubernetes-cluster-using-glooctl) (**recommended**)
-* [Ingress](#install-the-gloo-ingress-controller-to-your-kubernetes-cluster-using-glooctl)
-* [Knative](http://localhost:1313/installation/kubernetes/quick_start/#install-the-gloo-knative-cluster-ingress-to-your-kubernetes-cluster-using-glooctl)
+To install on any platform run the following.
```

Comment:
Right now we are listing the different methods to get `glooctl` one after another, which is not very clear. I think it would be better to have bullet points that describe each one:

- Via homebrew/gofish
- Via curl
- Via github release page
-